pkg/frontend/querymiddleware/roundtrip_test.go

```diff
@@ -43,6 +44,7 @@ func TestRangeTripperware(t *testing.T) {
 			http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 				var err error
 				if r.RequestURI == query {
+					w.Header().Set("Content-Type", jsonMimeType)
```

Comment:
Why is this required? I think in case of missing `Content-Type` we should fallback to JSON, so we shouldn't necessarily add this in all tests. Am I missing anything?

Reply:
Queriers (or, more specifically, the Prometheus API code we use) have always set `Content-Type: application/json`, so I favoured making the tests more realistic over adding a fallback that should never be used.